docs/src/data_documentation.md

```diff
@@ -237,14 +237,14 @@ negative flows indicate flow from destination to origin zone.
 
 This file includes parameters to characterize model temporal resolution to approximate annual grid operations, electricity demand for each time step for each zone, and cost of load shedding. Note that GenX is designed to model hourly time steps. With some care and effort, finer (e.g. 15 minute) or courser (e.g. 2 hour) time steps can be modeled so long as all time-related parameters are scaled appropriately (e.g. time period weights, heat rates, ramp rates and minimum up and down times for generators, variable costs, etc).
 
-###### Table 4: Structure of the Load\_data.csv file
+###### Table 4: Structure of the Demand\_data.csv file
 ---
 |**Column Name** | **Description**|
 | :------------ | :-----------|
 |**Mandatory Columns**|
-|Voll |Value of lost load in $/MWh.|
-|Demand\_Segment |Number of demand curtailment/lost load segments with different cost and capacity of curtailable demand for each segment. User-specified demand segments. Integer values starting with 1 in the first row. Additional segements added in subsequent rows.|
-|Cost\_of\_Demand\_Curtailment\_per\_MW |Cost of non-served energy/demand curtailment (for each segment), reported as a fraction of value of lost load. If *Demand\_Segment = 1*, then this parameter is a scalar and equal to one. In general this parameter is a vector of length equal to the length of Demand\_Segment.|
+|Voll |Value of lost consumption in $/MWh.|
+|Demand\_Segment |Number of demand curtailment/unserved demand segments with different cost and capacity of curtailable demand for each segment. User-specified demand segments. Integer values starting with 1 in the first row. Additional segements added in subsequent rows.|
```

Comment:
Value of "lost load" rather than "lost consumption" might just be the stock phrase here. Trying things out.

Reply:
Value of lost load is definitely the industry terminology. You could also call it "cost of unserved energy" if we are trying to generalize to more than just electricity, but VoLL is definitely the most used term.